lib/kernel-manager.coffee

```diff
@@ -176,3 +177,6 @@ module.exports = KernelManager =
 
     destroy: ->
         _.forEach @runningKernels, (kernel) -> kernel.destroy()
+
+    getStartupCode: (language) ->
+        return @getConfigJson('startupCode')[language] + ' \n'
```

Comment:
If no startup script is defined, this function returns the string 'undefined\n'.